src/Test/Utilities/Portable/FX/PinnedBlob.cs

```diff
@@ -1,34 +1,38 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
 using System;
+using System.Buffers;
 using System.Collections.Immutable;
 using System.Runtime.InteropServices;
 using Microsoft.CodeAnalysis;
 using Roslyn.Utilities;
 
 namespace Roslyn.Test.Utilities
 {
-    internal class PinnedBlob : IDisposable
+    internal unsafe class PinnedBlob : IDisposable
```

Comment:
💡 Can we put `unsafe` on the _members_ instead of the type? I try to avoid unnecessarily-wide scoping of this keyword.